source/administration/changelog.md

```diff
@@ -4,11 +4,82 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
+## Release v5.9
+
+Release Date 2019-03-16
+
+- Mattermost v5.9.0 contains low to medium level security fixes. [Upgrading](http://docs.mattermost.com/administration/upgrade.html) is highly recommended. Details will be posted on our [security updates page](https://about.mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://www.mattermost.org/responsible-disclosure-policy/).
+
+### Bug Fixes
+
+ - Fixed an issue where emoji reactions did not appear on posts right away.
+ - Fixed an issue where email notification setting in the webapp was out of sync with the mobile apps.
+ - Fixed an issue where a broken image displayed on login page if custom branding was enabled but no image had been uploaded.
+ - Fixed an issue where the emoji `Recently Used` cleared entirely after logging out and back in.
+ - Fixed an issue where at-channel, at-all, at-here followed by a period were not highlighted as mentions.
+ - Fixed an issue where emoji not included in our list of text-based emoji were not rendered as jumboemoji.
+ - Fixed an issue where the Mattermost icon was pixelated in bookmark rendering on Google Chrome.
+ - Fixed an issue where **System Console > Users** page had broken User Interface on narrow screens.
+ - Fixed an issue where at-channel alert showed incorrect number of timezones.
+ - Fixed an issue where leading whitespace with emoji affected emoji size so that they didn't render as jumboemoji.
+ - Fixed an issue where the admin panel graphs did not load smoothly.
+ - Fixed an issue with inconsistent formatting in page header on **System Console > Notifications > Mobile Push**.
+ - Fixed an issue where invite tokens with a 48-hour expiry were cleaned up after 24 hours.
+ - Fixed an issue where a blank screen appeared when opening a Group Message channel from "More" modal using Enter key.
+ - Fixed an issue where Zoom plugin caused link metadata code to print warnings.
+ 
+### Compatibility
+
+#### Breaking Changes
+
+ - Users with an outdated mobile version (such as a custom built app) might experience issues if the config.json is based on the new default.json ``DisableLegacyMFA``.
+ - Mobile app version 1.13+ is required for servers 5.4+
+
+### config.json
+
+One setting option was added to `config.json`. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
+ 
+#### Changes to Team Edition and Enterprise Edition:
+ 
+ - Under ``"ServiceSettings"`` in ``config.json``:
+    - Added ``"DisableLegacyMFA": true,`` to disable the checkMFA endpoint by default.
+
+### API Changes
+
+### Websocket Event Changes
+
+### Database Changes
+
+### Known Issues
+
+ - On a server using a subpath, the URL opens a blank page if the system admin changes the Site URL in the System Console UI. The system admin should restart the server to fix it.
```

Comment:
Hm, feels like this page is the same every release..wondering if anything should be updated here?